demos/enhancements/load-cora.ipynb

```diff
@@ -0,0 +1,210 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Stellargraph example: Load the CORA citation network"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Import stellar:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "import os\n",
+    "\n",
+    "from stellargraph.core.edge_data import edge_data as to_edge_data, IS_DIRECTED"
```

Comment:
If this is going to be imported under this name in the only example that uses it, maybe it could be renamed to `to_edge_data` in the main codebase?

Reply:
Funnily enough, it used to be `to_edge_data()` then `edge_data()` and now `to_edge_data()` again!